### PR TITLE
Imports Styling

### PR DESCRIPTION
**Pull request by helen-li**
# Description

Complete import functionality with updated Figma designs. Alert window just got cut off in last screenshot. 
<div>
Screen Shot 2023-04-28 at 4 56 26 PM
Screen Shot 2023-04-28 at 5 22 40 PM
Screen Shot 2023-04-28 at 5 22 54 PM
</div>

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have run `npm run lint:fix`
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by Manal-jpg — COMMENTED** [review]
Everything looks good to me.

**Review by annie951977 — APPROVED** [review]
Looks great, Helen :D

File: client/src/utils/HelperUtils.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    const GB = MB * 1024;
+
+    if (size < KB) {
+        return `${size} B`;
+    } else if (size < MB) {
+        return `${(size / KB).toFixed(2)} KB`;
+    } else if (size < GB) {
+        return `${(size / MB).toFixed(2)} MB`;
+    } else {
+        return `${(size / GB).toFixed(2)} GB`;
```
Comment (edwardhuahan) [inline client/src/utils/HelperUtils.ts:13]: This function feels wrong but it's very readable so it's I think it's good

**Review by edwardhuahan — APPROVED** [review]
Looks great, everything is very readable and I can understand it

**Pull request merged**